config/opta-k8s-service-helm/values.yaml

```
@@ -17,6 +17,7 @@ httpPort: null
 serviceAnnotations: {}
 keepPathPrefix: false
 persistentStorage: []
+ingress_extra_annotations: {}
```

```suggestion
ingressExtraAnnotations: {}
```

For consistency with other values, the value field should use camel case.